sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -832,7 +832,7 @@ object RapidsConf {
     .booleanConf
     .createWithDefault(true)
 
-  val USE_ARROW_OPT = conf("spark.rapids.arrowCopyOptmizationEnabled")
+  val USE_ARROW_OPT = conf("spark.rapids.arrowCopyOptimizationEnabled")
```

Comment:
Shouldn't there be a corresponding `configs.md` update that corresponds to this change?

Reply:
its an internal config  - mostly a test config to easily switch implementations, but I also figured it was a backup if something was going wrong could turn off the arrow stuff.

Reply:
ah right, I missed this was marked internal.